docs/t-sql/statements/create-function-transact-sql.md

```diff
@@ -737,17 +737,18 @@ WITH EMP_cte(EmployeeID, OrganizationNode, FirstName, LastName, JobTitle, Recurs
         -- Get the initial list of Employees for Manager n
         SELECT e.BusinessEntityID, e.OrganizationNode, p.FirstName, p.LastName, e.JobTitle, 0
```

Comment:
```suggestion
        SELECT e.BusinessEntityID, OrganizationNode = ISNULL(e.OrganizationNode, CAST('/' AS hierarchyid)) 
        , p.FirstName, p.LastName, e.JobTitle, 0
```

Reply:
Hi  @WilliamDAssafMSFT, this is good, the *ISNULL* function is better than *coalesce* here.